### PR TITLE
[4.4] Fix display of total votes

### PR DESCRIPTION
**Pull request by Quy**
Pull Request for Issue #43728. Regression of #42934.

### Summary of Changes
Move the number of votes outside of the visually-hidden section so it can be visible.

### Testing Instructions
Enable Voting in Articles component configuration.
Vote for one or more articles.

Edit `Content -Vote` plugin.
Toggle `Number of Votes` to `Show`.

### Actual result BEFORE applying this Pull Request
Total number of votes is not visible.

### Expected result AFTER applying this Pull Request
Total number of votes is visible.

### CODE REVIEW

File: plugins/content/vote/tmpl/rating.php [hunk 1]
```diff
@@ -81,15 +81,15 @@
...
             <meta itemprop="ratingCount" content="<?php echo $rcount; ?>">
             <meta itemprop="worstRating" content="1">
         </p>
-        <?php if ($this->params->get('show_total_votes', 0)) : ?>
-            <?php echo Text::sprintf('PLG_VOTE_TOTAL_VOTES', $rcount); ?>
-        <?php endif; ?>
     </div>
+    <?php if ($this->params->get('show_total_votes', 0)) : ?>
+        <?php echo Text::sprintf('PLG_VOTE_TOTAL_VOTES', $rcount); ?>
+    <?php endif; ?>
```
Comment (Quy) [inline plugins/content/vote/tmpl/rating.php]:
```suggestion
        <?php if ($this->params->get('show_total_votes', 0)) : ?>
            <?php echo Text::sprintf('PLG_VOTE_TOTAL_VOTES', $rcount); ?>
        <?php endif; ?>
```

**Review by chmst — COMMENTED** [review]
The whole block after    ` <?php if ($rcount) : ?> `needs to be indented

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 9ed4867123a277c32a7ca926c4d37b545c30287e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43740">issues.joomla.org/tracker/joomla-cms/43740</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 9ed4867123a277c32a7ca926c4d37b545c30287e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43740">issues.joomla.org/tracker/joomla-cms/43740</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43740">issues.joomla.org/tracker/joomla-cms/43740</a>.</sub>

**Comment by brianteeman** [comment]
> The whole block after `<?php if ($rcount) : ?>`needs to be indented

How can this be rtc while this is still outstanding to be fixed

**Comment by richard67** [comment]
> > The whole block after `<?php if ($rcount) : ?>`needs to be indented
> 
> How can this be rtc while this is still outstanding to be fixed

To me the code style seems to be ok, and PHPCS seems to be ok with it, too.

**Comment by richard67** [comment]
> Thanks for this PR and testing.
> 
> Just a question. Who or what ensures that this PR also applies to J5?

@sandewt Same as always for all bug fixes in 4.4-dev: When the PR has been merged into 4.4-dev, release managers of 5.1 will merge it up into 5.1-dev and so on.

**Comment by brianteeman** [comment]
> > > The whole block after `<?php if ($rcount) : ?>`needs to be indented
> > 
> > 
> > How can this be rtc while this is still outstanding to be fixed
> 
> To me the code style seems to be ok, and PHPCS seems to be ok with it, too.

when you look at the entire file (not just the snippet shown in github) then you can see that @chmst is correct

**Comment by richard67** [comment]
@Quy Thanks for fixing the CS.

Leaving RTC status as the change after setting RTC was just a code style change, so previous human tests are still valid.

@chmst Thanks for review, and thanks @ChristineWk and @viocassel for testing.

**Pull request merged**